samples/msgext-northwind-inventory-python/lab/Exercise 02 - Run sample app.md

```diff
@@ -0,0 +1,235 @@
+# Building Message Extensions for Microsoft Copilot for Microsoft 365
+
+TABLE OF CONTENTS
+
+* [Welcome](./Exercise%2000%20-%20Welcome.md) 
+* [Exercise 1](./Exercise%2001%20-%20Set%20up.md) - Set up your development Environment 
+* Exercise 2 - Run the sample as a Message Extension **(THIS PAGE)**
+* [Exercise 3](./Exercise%2003%20-%20Run%20in%20Copilot.md) - Run the sample as a Copilot plugin
+* [Exercise 4](./Exercise%2004%20-%20Add%20a%20new%20command.md) - Add a new command
+* [Exercise 5](./Exercise%2005%20-%20Code%20tour.md) - Code tour
+
+## Exercise 2 - Run the sample as a Message Extension
+
+## Step 1 - Set up the project for first use
```

Comment:
There is no action for the action to perform in this step as the connection string is already prefilled for the user, consider removing the step.

Reply:
Fixed